app/app.go

```diff
@@ -309,26 +280,26 @@ func NewApp(
 	return app
 }
 
-func (app *AkashApp) initChainer(
-	ctx sdk.Context, req abci.RequestInitChain) abci.ResponseInitChain {
-	var genesisState simapp.GenesisState
-	app.cdc.MustUnmarshalJSON(req.AppStateBytes, &genesisState)
-
-	return app.mm.InitGenesis(ctx, genesisState)
-}
+// Name returns the name of the App
+func (app *AkashApp) Name() string { return app.BaseApp.Name() }
 
-// application updates every begin block
-func (app *AkashApp) beginBlocker(
-	ctx sdk.Context, req abci.RequestBeginBlock) abci.ResponseBeginBlock {
+// BeginBlocker application updates every begin block
+func (app *AkashApp) BeginBlocker(ctx sdk.Context, req abci.RequestBeginBlock) abci.ResponseBeginBlock {
```

Comment:
why are you making these public?

Reply:
One reason is, it helps in simapp tests

Reply:
that's the standard of the SDK framework. It doesn´t make much difference since the function calls a private function. Otherwise, you can call the module manager, `mm`, directly so there's no need to have a private function call another private function imho.

Reply:
Exporting symbols comes with many costs.  Please don't export any symbols unless absolutely necessary.